CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing to caMicroscope
+We love your input! We want to make contributing to this project as easy and transparent as possible, whether it's:
+
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+- Becoming a maintainer
+
+## We Develop with Github
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
+
+## We Use [Github Flow](https://guides.github.com/introduction/flow/index.html), So All Code Changes Happen Through Pull Requests
+Pull requests are the best way to propose changes to the codebase (we use [Github Flow](https://guides.github.com/introduction/flow/index.html)). We actively welcome your pull requests:
+
+1. Fork the repo and create your branch from `master`.
+2. If you've added code that should be tested, add tests.
+3. If you've changed APIs, update the documentation.
+4. Ensure the test suite passes.
+5. Make sure your code lints.
```

Comment:
```suggestion
5. Make sure your code passes the lint tests, by running `npm run lint-fix` ( after installing the dependencies using `npm install` ). 
```

Reply:
@cjchirag7, I think so this has been mentioned in a seperate [section](https://github.com/Vedant1202/caMicroscope/blob/contrib-PR/CONTRIBUTING.md#use-a-consistent-coding-style). I'll reference to that section though